packages/forma-36-react-components/src/components/Button/Button.tsx

```diff
@@ -48,32 +49,22 @@ export interface ButtonProps {
   isActive?: boolean;
 }
 
-const defaultProps: Partial<ButtonProps> = {
-  loading: false,
-  isFullWidth: false,
-  indicateDropdown: false,
-  disabled: false,
-  testId: 'cf-ui-button',
-  buttonType: 'primary',
-  type: 'button',
-};
-
 export const Button = (props: ButtonProps) => {
   const {
     className,
     children,
     icon,
-    buttonType,
+    buttonType = 'primary',
     size,
-    isFullWidth,
+    isFullWidth = false,
     onBlur,
-    testId,
+    testId = 'cf-ui-button',
     onClick,
-    loading,
-    disabled,
-    indicateDropdown,
+    loading = false,
+    disabled = false,
+    indicateDropdown = false,
     href,
-    type,
+    type = 'button',
```

Comment:
I think we need to destructure the props in the argument of the function otherwise Storybook does not pick the default values so the documentation looks like this:

![Screenshot 2021-01-06 at 10 02 09](https://user-images.githubusercontent.com/6597467/103749986-7e10fb00-5006-11eb-9a8f-bbd8be471763.png)

The Accordion, for example, is coded like this and the default values show up in the prop table:
```jsx
({ optionalProp: 'defaultValue' }: Props) => { ... }
```

Reply:
but this is something we can do every time we need to change something in some component!